apps/network-alerts/js/attention/attention.js

```diff
@@ -76,9 +77,25 @@ function onVisibilityChange() {
   }
 }
 
-function renderForm() {
+function notify() {
+  Notify.notify().then((stopFunc) => {
+    if (document.hidden) {
+      stopFunc.stop();
```

Comment:
Not sure if it's proper to stop while hidden(like press power key or home button), since the document didn't define this use case. Maybe we can ask Carol about it?